src/Components/Admin/EnvironmentEditor/environment.js

```diff
@@ -64,6 +69,38 @@ const useStyles = makeStyles((theme) => ({
         backgroundColor: "#E2E5ED",
         borderRadius: "12px",
     },
+    search: {
```

Comment:
What is `backgroundColor` here? If it's used elsewhere, can you put this into the colours file?